src/sdk/sql_cluster_router.cc

```diff
@@ -4143,10 +4162,89 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::ExecuteShowTableStat
                         std::to_string(mem_bytes), std::to_string(disk_bytes), std::to_string(partition_num),
                         std::to_string(partition_unalive), std::to_string(replica_num), offline_path, offline_format,
                         offline_deep_copy});
-    });
+        if (all) {
+            data.back().push_back(error_msg);
+        }
+    }
 
     // TODO(#1456): rich schema result set, and pretty-print numberic values (e.g timestamp) in cli
-    return ResultSetSQL::MakeResultSet(GetTableStatusSchema(), data, status);
+    return ResultSetSQL::MakeResultSet(all ? GetTableStatusAllSchema() : GetTableStatusSchema(), data, status);
```

Comment:
two schema ?

Reply:
I keep these two first. Let's discuss whether it is ok to reuse the `show table status` command and extend the result with an extra `warnings` column.

If no db is selected, show all tables including the hidden ones. 

If it is ok, only need to keep the bigger schema and some logics can also be removed.